lib/helpers/run-processors.js

```diff
@@ -6,6 +6,9 @@ module.exports = function runProcessors(contents, processor, file) {
   const result = {file};
 
   const newContents = processors.reduce((contents, processor) => {
+    if (processor.length === 2) {
+      return processor(contents, file);
+    }
     return processor(contents);
```

Comment:
```suggestion
    return processor(contents, file);
```

I think specifically checking for number of arguments the function expects may potentially be a breaking change, if someone is using a processor which expects (but doesn't use) 2 arguments at the moment.

This could also be an issue if passing the 2nd argument in directly though. So I'll probably release this as a breaking change/major release anyway. 

But if we do that, then might as well always pass the file as a parameter imo.

Reply:
Sure thing. To be honest JS is not the language I'm most familiar with. Thanks for the explanation!